src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -326,6 +327,10 @@ private static bool CompileProject(ProjectContext context, string configuration,
                 return false;
             }
 
+            var assemblyInfo = Path.Combine(intermediateOutputPath, $"dotnet-compile.assemblyinfo.cs");
```

Comment:
Also, as we discussed earlier, it's gross but for now, have `dotnet-compile-csc` load up all the source file syntax trees and scan to see if any of these attributes already exists. Ideally, this would be something `csc` would do for us.